v1.1/licensing.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Licensing
+summary: Request and set trial and enterprise license keys for CockroachDB
+toc: false
+---
+
+This page shows you how to request and set the trial and enterprise license keys for CockroachDB.
+
+<div id="toc"></div>
+
+CockroachDB distributes a single binary that contains both core and [enterprise features](https://www.cockroachlabs.com/pricing/). You can use core features without any license key. However, to use the enterprise features, you need either a trial or an enterprise license key.
+
+## Types of Licenses
+
+You can request a trial license or an enterprise license:
+
+- Trial license: A trial license enables you to try out CockroachDB enterprise features for 30 days. After 30 days, you can upgrade to the enterprise license by [requesting](licensing.html#request-a-trial-or-an-enterprise-license-key) and [setting](licensing.html#set-the-trial-or-enterprise-license-key) an enterprise license key.
```

Comment:
This is perfectly fine, but I think a table would be a more visually effective way to present this:

```
Type | Description
-----|------------
**Trial License** | xxx
**Enterprise License** |  xxx
```

Reply:
Fixed.